spec/features/translations_spec.rb

```diff
@@ -71,6 +71,18 @@
       expect(page).not_to have_link "Español"
     end
 
+    scenario 'Change value of a translated field to blank' do
+      visit @edit_milestone_url
+
+      fill_in 'budget_investment_milestone_description_en', with: ''
+
+      click_button "Update milestone"
+      expect(page).to have_content "Milestone updated successfully"
+
+      expect(page).to have_content "Milestone updated successfully"
```

Comment:
A very small thing: this line is now duplicated.

EDIT: If changing this line forces you to rebase all branches depending in this one, I think it's OK to leave it like this.

Reply:
:/ Okay, I'll make a note of it to fix later (either if something more serious comes up and I have to rebase anyway, or after this gets merged)